shared/settings/delete-confirm/container.js

```diff
@@ -2,14 +2,30 @@
 import React, {Component} from 'react'
 import {TypedConnector} from '../../util/typed-connect'
 import {navigateUp} from '../../actions/router'
+import {HOCTimers} from '../../common-adapters'
 import DeleteConfirm from './index'
-import {deleteAccountForever} from '../../actions/settings'
+import {setAllowDeleteAccount, deleteAccountForever} from '../../actions/settings'
 
 import type {TypedDispatch} from '../../constants/types/flux'
 import type {TypedState} from '../../constants/reducer'
+import type {TimerProps} from '../../common-adapters/hoc-timers'
 import type {Props} from './index'
 
-class DeleteConfirmContainer extends Component<void, Props, void> {
+class DeleteConfirmContainer extends Component<void, Props & TimerProps, void> {
+  componentWillMount () {
```

Comment:
I'm far past my 2 story points at this point... let's go with this first and we can continue to refine it.

Another thing option is to add a checkbox to the confirm screen so that users must acknowledge that their next click is permanent. Might kill 2 birds with one stone and remove the need for the timer.